crates/ika-core/src/dwallet_mpc/crytographic_computation/mpc_computations/reconfiguration.rs

```diff
@@ -105,7 +125,7 @@ impl ReconfigurationV2PartyPublicInputGenerator for ReconfigurationV2Secp256k1Pa
                         upcoming_encryption_keys_per_crt_prime_and_proofs.clone(),
                         current_tangible_party_id_to_upcoming(current_committee, upcoming_committee)
                             .clone(),
-                        bcs::from_bytes(&network_dkg_public_output)?,
+                        public_output.into(),
```

Comment:
```suggestion
                        public_output,
```

Reply:
No `.into()` needed. Maybe you are in an old version, use the latest from `main`.